index.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module "win-ca";
```

Comment:
Why is this necessary?

Reply:
This library does not have types, this causes an error when using TS. declaring the module here, will prevent lint errors.

Reply:
Could we solve it by changing the `import * as ca ...` statement to `const ca = require('win-ca');`?

Reply:
suggestion followed, removed index.d.ts as no longer required